### ISSUE TITLE
Admissions content accessibility audit

### ISSUE THREAD

**Issue by Computerrat** [phsc-edu/2018#333]
### Please check the following links for accessibility problems. ###
 
https://admissions.phsc.edu/help/faq/graduation/commencement-ceremony-procedural-information
![commencement ceremony procedural information _ admissions and student records - google chrome 2018-08-06 09 46 36](https://user-images.githubusercontent.com/31025634/43726446-279e5c1a-996d-11e8-9a1d-2410d1b863e5.png)

--------------

https://admissions.phsc.edu/help/faq/transfer-students/if-i-am-transfer-student-who-has-not-successfully-completed-college
![if i am a transfer student who has not successfully completed a college-level english or mathematics course what will i need_ _ admissions and student records - google chrome 2018-08-06 09 58 50](https://user-images.githubusercontent.com/31025634/43726500-52acd710-996d-11e8-8321-8211e262262f.png)

----------------

https://admissions.phsc.edu/transcripts/submission-official-transcripts
![submission of official transcripts _ admissions and student records - google chrome 2018-08-06 09 35 58](https://user-images.githubusercontent.com/31025634/43726548-6feac7ce-996d-11e8-90d8-a9c7ee904d90.png)

**Comment by C13L0** [comment]
@curtiskphsc Please correct the three accessibility issues as documented above.

**Comment by curtiskphsc** [comment]
@C13L0 @Computerrat This is the same scenario. It is not enough for us to simply say "This is not accessible" We need to be very clear in how we communicate this to the content creators otherwise they will simply keep repeating the same mistakes. Also, we don't want to simply make the changes ourselves or they will not learn.
What is the ADA compliant way to include a hyperlink to an external web page in the copy?
Why can't they place an email in the copy as plain text? What is the ADA compliant way to include an email address in the copy?
Why can't they use all caps to highlight important text? How do they highlight important text without using all caps or using bold?

We should be quoting these from a database of information, and again, we need to be on the same page and speaking the exact same language to avoid confusion among our stakeholders.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc Please correct the three accessibility issues as documented above.

**Comment by curtiskphsc** [comment]
@C13L0 @Computerrat ready for review

**Comment by C13L0** [comment]
@Computerrat Please review changes for accessibility and close ticket once verified.

**Comment by Computerrat** [comment]
Verified all corrections made
